scripts/dig.js

```diff
@@ -0,0 +1,27 @@
+var b = pizza.open('https://toolbox.googleapps.com/apps/dig/');
+
+pizza.sleep(1000);
+// b.waitPageLoad();  // script has been getting hung up
+
+var domains = ['walmart.com', 'amazon.com'];
+
+b.type('#domain', domains[Math.floor((Math.random() * domains.length - 1) + 1)]);
```

Comment:
Pizzascript has a handy function for selecting a random item from a list:

utils.randomElement(domains);